scripts/gen_keys_poa_validator

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
This script is a wrapper around subkey, it first generates an account, then it generates a p2p key. I don't see the value this wrapper provides.

Assuming this is a user facing tool. I propose we let the validator invoke subkey directly if necessary. This will grant them greater flexibility and give them one less script to worry about.

Reply:
We can mention that in the instructions and let them use subkey if they wish to.

Reply:
Good idea. What do you think about removing this script so they have one less thing to think about?

Reply:
I suggest having the script as well and let them choose, if we realize during testnet that folks are not using it, we get rid of that